src/test/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortFilterRecordUnitTest.java

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import org.apache.avro.generic.GenericRecord;
+import org.broadinstitute.hellbender.GATKBaseTest;
+
+import org.broadinstitute.hellbender.utils.bigquery.AvroFileReader;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ExtractCohortFilterRecordUnitTest extends GATKBaseTest {
+
+    // TODO refactor - this is copied from ExtractCohortRecordUnitTest
+    private static GenericRecord getSingleRecordFromAvroFile (String inputFileName) {
+        final AvroFileReader avroFileReader = new AvroFileReader(inputFileName);
+        return avroFileReader.next();
+    }
+
+    // load in test data
+    private static final String testFileDir = "src/test/resources/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortFilterRecord/";
```

Comment:
tried using `getToolTestDataDir()` but get a "non-static method cannot be referenced from static context" complaint

Reply:
make this and the above method non-static (mostly so you can call that other instance method)